app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -191,6 +197,11 @@ public View inflateLayout(LayoutInflater inflater, @Nullable ViewGroup container
             initialiseNormalBrowserUi();
         }
 
+
+        SPEED_MODE = Settings.getInstance(this.getContext()).shouldUseSpeedMode();
+
+        setBlockingEnabled(SPEED_MODE);
+
```

Comment:
I think we don't need this because we have all things setup during Creation of Webview and every time the preference is changed?